.eslintrc.js

```diff
@@ -0,0 +1,291 @@
+              fires: 'emits',
+            },
+            requireParamDescription: false,
+            requireReturn: true,
+            requireReturnType: true,
+            requireReturnDescription: false,
+          },
+        ],
+        'require-jsdoc': [
+          'warn',
+          {
+            require: {
+              FunctionDeclaration: true,
+              MethodDefinition: true,
+              ClassDeclaration: true,
+              ArrowFunctionExpression: false,
+              FunctionExpression: true,
+            },
+          },
+        ],
+      },
+    },
+    {
+      files: ['*.js'],
+      rules: {
+        '@typescript-eslint/no-var-requires': 1,
+        'valid-jsdoc': [
+          'error',
+          {
+            prefer: {
+              arg: 'param',
+              argument: 'param',
+              return: 'returns',
+              virtual: 'abstract',
+              fires: 'emits',
+            },
+            requireParamDescription: false,
+            requireReturn: true,
+            requireReturnType: true,
+            requireReturnDescription: false,
+          },
+        ],
+        'require-jsdoc': [
+          'warn',
+          {
+            require: {
+              FunctionDeclaration: true,
+              MethodDefinition: true,
+              ClassDeclaration: true,
+              ArrowFunctionExpression: false,
+              FunctionExpression: true,
+            },
+          },
+        ],
+        'tsdoc/syntax': 'off',
+        'class-methods-use-this': 0,
+        'no-multi-assign': 0,
+        'no-promise-executor-return': 0,
+        'no-restricted-exports': 0,
+      },
+    },
+    {
+      files: ['packages/**/test/**/*.[tj]s', 'packages/**/*-spec/**/*.[tj]s'],
+      env: {
+        mocha: true,
+      },
+      rules: {
+        'func-names': 0,
+        'no-invalid-this': 0,
+        'require-jsdoc': 0,
+      },
+    },
+    {
+      files: ['wdio.helpers.d/**/*.js', 'docs/samples/**/test/wdio/**/*.js'],
+      globals: {
+        browser: false,
+        browserFirefox: false,
+        browserChrome: false,
+        $: false,
+        $$: false,
+        step: false,
+        describe: false,
+        before: false,
+        beforeAll: false,
+        beforeEach: false,
+        after: false,
+        afterAll: false,
+        afterEach: false,
+        it: false,
+      },
+    },
+    {
+      files: ['packages/@webex/xunit-with-logs'],
+      rules: {
+        'func-names': 0,
+        'no-console': 0,
+      },
+    },
+    {
+      files: ['packages/**/bin-*/**'],
+      rules: {
+        'no-sync': 0,
+      },
+    },
+  ],
+
+  rules: {
+    'comma-dangle': ['error', 'only-multiline'],
+    quotes: [
+      'error',
+      'single',
+      {
+        avoidEscape: true,
+      },
+    ],
+    'tsdoc/syntax': 'warn',
+    'object-curly-spacing': 0,
+    'no-shadow': 0,
+    '@typescript-eslint/no-shadow': 1, // TODO: Fix this in a later PR
```

Comment:
I think you moved this next to the wrong no-shadow

Reply:
though I don't know why this no-shadow is 0

Reply:
should be fixed now i removed it